packages/contracts/contracts/L2/predeploys/OVM_L1BlockNumber.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import { iOVM_L1BlockNumber } from "./iOVM_L1BlockNumber.sol";
+import { L1Block } from "../L1Block.sol";
+
+contract OVM_L1BlockNumber is iOVM_L1BlockNumber {
+    address public l1BlockAddress;
```

Comment:
Perhaps make this internal? Will be exposing more public API that will become maintenance burden over time